docs/manual/guides/install-test-versions.de.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Testversionen installieren"
+description: "Anleitung zur Installation von Entwicklungsversionen und Release Candidates."
+url: "anleitungen/testversionen-installieren"
+weight: 20
+---
+
+
+Das Entwicklerteam von Contao hält sich an einen festgelegten [Release-Plan][releasePlan]
+bei der Entwicklung neuer Versionen (bzw. dem Support bestehender). Vor jeder geplanten
+neuen Version gibt es einen gewissen Zeitraum in dem aktiv an dieser Version entwickelt
+wird. Neue Features werden hinzugefügt, bis ein gewisser Zeitpunkt erreicht ist.
+Danach werden die ersten »Release Candidates« veröffentlicht. Wenn während der Tests
+keine weiteren Probleme gefunden werden, wird die erste stabile Version veröffentlicht.
+
+Auch wenn Contao Unit Testing einsetzt können immer unvorhergesehen Probleme und
+Fehler auftauchen, nachdem neue Features hinzugefügt, oder bestehende Dinge
+geändert wurden. Daher müssen neue Versionen von tatsächlichen Benutzern getestet 
+werden. Als Open Source Software kann Contao dabei aber vom Engagement der Community 
+profitieren.
+
+Im Folgenden wird erklärt wie man neue Versionen von Contao selbst installieren
+und daher testen kann. Entweder durch die Installation eines Release Candidates,
+oder sogar einer aktuellen Entwicklerversion.
```

Comment:
Maybe we should write this as a complete sentence (or put it in brackets instead).